Sniffs/PHP/ForbiddenNamesAsDeclaredSniff.php

```diff
@@ -183,7 +207,11 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
                 $tokens[$stackPtr]['type'],
             );
 
-            $phpcsFile->addError($error, $stackPtr, $errorCode, $data);
```

Comment:
Should the error message be the same for both ? or should the `soft` reserved words say that (they are soft reserved) ?